kinoml/workflows/chembl22_cnn.py

```diff
@@ -0,0 +1,188 @@
+'''
+This script uses the pre-processed data file df_reduced.pkl that is generated from the ChEMBL22 data set.
+
+Type of target used: single-target (i.e. ligand-based model)
+Type of featurizer for the ligand: One Hot Encoding of SMILES
+Type of model: a Convolutional Neural Network (CNN) using Pytorch
+
+Please run in terminal the following command:
+$ export PYTHONPATH=$(dirname $(pwd))
+
+And check the PYTHONPATH with: 
+$ echo $PYTHONPATH
+
+'''
+# Imports
+import pandas as pd
+import numpy as np
+from pathlib import Path
+from sklearn.model_selection import train_test_split, KFold
+from sklearn.metrics import accuracy_score, confusion_matrix
+
+import torch
+import torch.nn as nn
+import torch.optim as optim
+
+from ml.models_pytorch import CNN
+from features.ligand import OneHotSMILESFeaturizer
+
+
+if __name__ == '__main__':
+
+    # Load pre-processed data
+    p = Path(__file__).resolve()
+    data = pd.read_pickle(f'{p.parents[1]}/data/df_reduced.pkl')
```

Comment:
We need to figure out a way to use entry points for data files. At least a helper function, applicable all across the library, should be used here (something like `get_data_file('df_reduced.pkl')`).

Reply:
By the way, we should try to parameterize the input file in case somebody wants to try a different dataset (with similar data in it), without having to patch this one.